frontend/src/lang/es.ts

```diff
@@ -199,6 +199,7 @@ export const es: LanguageConfig = {
           city: "La ciudad tiene un formato incorrecto",
           county: "El formato del condado es incorrecto",
           country: "El formato del país es incorrecto",
+          // TODO: update translation - "incorrect" -> "required"
           race: "La raza tiene un formato incorrecto",
```

Comment:
@Rebecca-LM ☝️

Reply:
@nathancrtr sorry if I missed it, but what's the English version of this? I want to be sure it's capturing it correctly in Spansih.

Reply:
The English version is `Race is required`

Reply:
@alismx the spanish talks about an incorrect format though. what's the english version for that part?

Reply:
@Rebecca-LM sorry, that was another question for you! In prod right now, that error message would be `Race is incorrectly formatted`. That struck me as confusing, though - race is a radio group, so "incorrectly formatted" input is impossible.

Changing that message to `Race is missing` would bring it up-to-date with the language we use for other omitted fields on the person form. Let me know if you think that error message is OK. Whatever we decide on will need a new translation as well.

Reply:
@nathancrtr is this a 2-line error toast or an inline message? i think we need to tell them to "Select an answer" or "Choose a response" (not sure what we say in other places..). if it's 2 lines that can go below. lmk!

Reply:
@Rebecca-LM here is the section of the translation config file where we define the error messages that may appear on the person form: https://github.com/CDCgov/prime-simplereport/blob/2bf162f14c0d46eddff8ca86165f9bf52fbb2538/frontend/src/lang/en.ts#L162-L203

The errors appear on a Toast alert with two lines, where the first line is one of the messages defined above and the second line is a generic "Please correct before submitting":

<img width="801" alt="Screen Shot 2022-01-27 at 1 37 01 PM" src="https://user-images.githubusercontent.com/27730981/151422461-47f9de32-dc9a-49c3-83cb-2d5d6e2848e2.png">

Reply:
Gotcha. In that case how about this: @nathancrtr 

**Race is required**
Please choose a response


**La raza es obligatoria**
Elije una respuesta

Reply:
Sounds great, thank you! 🤜 🤛